app/scripts/components/collections/overview.js

```diff
@@ -119,7 +119,7 @@ const CollectionOverview = React.createClass({
           <Link className='button button--small form-group__element--right button--green' to={`/collections/edit/${collectionName}/${collectionVersion}`}>Edit</Link>
           {lastUpdated(get(record, 'data.timestamp'))}
           {overview}
-          {errors.length ? <ErrorReport report={errors} /> : null}
+          {errors.length ? <ErrorReport report={errors} truncate={true} /> : null}
```

Comment:
why do we truncate the errors?

Reply:
In a lot of cases, especially for errors with stack traces, the error output is really long. So what `truncate` actually does is initially show a shorter version of the error message with a "show more" link. And when the "show more" link is clicked, it expands to show the whole error message. Also, I added logic so that it won't truncate/add the "show more" link unless the error message is long enough to need it